src/expandableCalendar/asCalendarConsumer.tsx

```diff
@@ -4,7 +4,7 @@ import hoistNonReactStatic from 'hoist-non-react-statics';
 import CalendarContext from './Context';
 
 
-function asCalendarConsumer<PROPS>(WrappedComponent: React.ComponentType<any>): React.ComponentClass<PROPS> {
+function asCalendarConsumer<PROPS>(WrappedComponent: React.ComponentType<any>, functional = false): React.ComponentClass<PROPS> {
```

Comment:
If we warp the functional component with `forwardRef` I think we won't need this, can you check?